packages/contact-center/task/src/IncomingTask/incoming-task.presentational.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {IncomingTaskPresentationalProps} from '../task.types';
+
+const IncomingTaskPresentational: React.FunctionComponent<IncomingTaskPresentationalProps> = (props) => {
+  const {currentTask, accept, decline, isBrowser, answered, ended, missed} = props;
+
+  if (!currentTask) {
+    return <></>; // hidden component
+  }
+
+  const callAssociationDetails = props.currentTask.data.interaction.callAssociatedDetails;
+  const {ani, dn, virtualTeamName} = callAssociationDetails;
+  const timeElapsed = ''; // TODO: Calculate time elapsed
+
+  return (
+    <div
+      data-testid="incoming-task-presentational"
+      style={{
+        border: '1px solid #ccc',
+        borderRadius: '8px',
+        padding: '16px',
+        width: '350px',
+        boxShadow: '0px 4px 8px rgba(0, 0, 0, 0.2)',
+        fontFamily: 'Arial, sans-serif',
+        backgroundColor: '#ffffff',
+        display: 'flex',
+        flexDirection: 'column',
+      }}
```

Comment:
Can we create an object styles at the top and then, use them here rather than having inline styles. Please see how this is done in station-login